spec/requests/api/tenant_quotas_spec.rb

```diff
@@ -0,0 +1,246 @@
+describe "tenant quotas API" do
+  let(:tenant) { FactoryGirl.create(:tenant) }
```

Comment:
oh ok, thanks for the explanation, it should have worked with `let!` then.
it is just a matter of taste, if you like it this way, I am perfectly fine with leaving tests as is